modules/pwa/assets/css/show-network-status.css

```diff
@@ -0,0 +1,7 @@
+#jetpack-network-status {
+    display: none;
+}
+
+body.jetpack__offline #jetpack-network-status {
+    display: block;
+}
```

Comment:
Minor drive-by nitpick. This, and the other files below, don't end with an empty newline.

Reply:
Fixed, thanks. Will push up this change soon.